### ISSUE TITLE
Events workshop banner unreadable with larger text sizes

### ISSUE THREAD

**Issue by frankcalise** [infinitered/ChainReactApp2023#216]
The banner that reads "Events for workshop ticket-holders" is not readable if the user has accessibility options such as Larger Text Size enabled

<img src="https://user-images.githubusercontent.com/374022/236910286-0bb8325c-123a-4e4f-80ed-fd4c49bb346b.png" width="300" />

**Comment by morganick** [comment]
Fixed in PR #222
